packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-methods/public/output.js

```diff
@@ -1,12 +1,12 @@
-var _computedKey, _dec, _dec2, _initProto;
+var _computedKey, _initProto;
+
+const dec = () => {};
 
-_dec = dec
 _computedKey = 'b'
-_dec2 = dec
 
 class Foo {
   static {
-    [_initProto] = babelHelpers.applyDecs(this, [[_dec, 2, "a"], [_dec2, 2, _computedKey]], []);
+    [_initProto] = babelHelpers.applyDecs(this, [[dec, 2, "a"], [dec, 2, _computedKey]], []);
```

Comment:
The new memoising approach generates less code when `dec` is considered `static`, e.g. a bound identifier.

Reply:
Oh good, this will probably be a common case.